Update/Fix some things (#13)
tiramisulabs/seyfert-docs: src/content/docs/commands/subcommands.mdx

</FileTree>

<Tabs>
	<TabItem label='parent.ts' icon="seti:typescript">

:::tip

With this file structure you can use `@Autoload()` and seyfert will add the commands for you.

:::

	```ts copy
import { type CommandContext, Declare, Command, Options } from "seyfert";
import { AddCommand } from "./create.command";
import { DeleteCommand } from "./delete.command";

@Declare({
	name: "account",
	description: "account command"

> })
> @Options([AddCommand, DeleteCommand]) // If the sub-commands are in the same directory can use  @Autoload()
> export default class AccountCommand extends Command {}

Reviewer: ```suggestion
})
// Being in the same folder with @AutoLoad() you can save this
@Options([AddCommand, DeleteCommand])
export default class AccountCommand extends Command {}
```

	name: "create",
	description: "create a new something"
})

> export class CreateCommand extends SubCommand {

Reviewer 2: When using autoload you must export default the SubCommand class!!
